### ISSUE TITLE
01 Server running on WSL2

### ISSUE THREAD

**Issue by kbfifi** [OpenInterpreter/01#161]
**Is your feature request related to a problem? Please describe.**
See #88 Running the server on W11, WSL2 runs into X11 issues although WSLg is enabled and working. As WSLg is supporting Wayland maybe it's not too difficult to support this

**Describe the solution you'd like**
I'd like to be able to run 01 server on a W11 machine

**Additional context**
-

**Comment by tyfiero** [comment]
Can you provide more information? Are you given any error messages? or is it the same as #88?

**Comment by kbfifi** [comment]
Yes it's the same message as #88 If figured that this project likely is missing WSL support so that's why I created this feature request

**Comment by rbrisita** [comment]
It was quite a journey trying to do audio through WSL2 the last time I tried. Leaving this link for reference:

https://github.com/microsoft/WSL/issues/5816

**Comment by trinsition** [comment]
> Yes it's the same message as #88 If figured that this project likely is missing WSL support so that's why I created this feature request

Did you happen to get anywhere with X11/WSL support for the 01 server?
